### ISSUE TITLE
Dispatch refresh does not include reading characters (question mark, dot)

### ISSUE THREAD

**Issue by hansmbakker** [Microsoft/botbuilder-tools#1018]
## Tool
Name: Dispatch
Version: 1.3.0
OS: Windows 10 build 17763

## Describe the bug
Dispatch CLI does not export characters like ? and . in `dispatch refresh`. However it seems to have an impact on the dispatch intent matching when the user does use this question mark when there is a similar question in another app.

Example (translated into English):
question "With whom should I sign a contract?" --> question in qna maker
utterance "should I sign a contract?" --> in a luis app to start a decision making dialog

`dispatch refresh` translates this into these utterances (without question marks):
- with whom should I sign a contract --> dispatch intent to qna maker
- should I sign a contract --> dispatch intent to luis app

Testing "should I sign a contract?" triggers the dispatch intent to qna maker, unless I manually add the question mark in dispatch

[bug]

**Comment by tsuwandy** [comment]
@hansmbakker , dispatch create command has the optional parameter "--dontReviseUtterance", which will keep the utterances from LUIS/QnA maker apps as is.  This behavior is added to make sure that that utterances with ? won't be heavily skewed toward QnA intent since most QnA questions end with ?.  

Dispatch documentation:
https://github.com/Microsoft/botbuilder-tools/tree/master/packages/Dispatch

Try using the param and see if it works better for your Dispatch.  Thanks!

**Comment by tsuwandy** [comment]
@hansmbakker, if don't mind sharing the source LUIS/QnA maker, that'll be great.  As Dispatch strips the character from both LUIS/QnA sources, the character '?' shouldn't have any weight in the prediction.

**Comment by hansmbakker** [comment]
@tsuwandy thank you very much for getting back to me!

We do not have the utterances in QnA Maker anymore so unfortunately I cannot quickly re-test this - we use LUIS exclusively for utterance recognition now to improve the performance on Dutch sentences containing jargon.

After we moved to this architecture, the QnA Maker team released a custom Dutch ranker to us as a preview, which seems to significantly improve prediction at least for short utterances. As we already shifted our architecture, I have not yet been able to test this new ranker with sentence-length utterances.

Let's close this issue for now, and it can be reopened should I (or somebody else) run into it again.
